src/librustc/dep_graph/dep_node.rs

```diff
@@ -625,6 +625,7 @@ define_dep_nodes!( <'tcx>
 
     // We use this for most things when incr. comp. is turned off.
     [] Null,
+    [] NormalizeTy,
```

Comment:
Could you please add this above Null, as thats a special thing?

Reply:
Sure!